test/functional/Java8andUp/src/org/openj9/test/attachAPI/AttachApiTest.java

```diff
@@ -200,5 +215,25 @@ protected boolean checkLoadAgentOutput(String errOutput,
 		}
 		return outputLines;
 	}
+
+	/**
+	 * Find an executable in the jdk directory and fail if it is missing.
+	 * @param commandRoot Base name of the command, without ".exe" 
+	 */
+	protected void getJdkUtilityPath(final String commandRoot) {
+		char fs = java.io.File.separatorChar;
+		final String commandWithSuffix = 
+				PlatformInfo.isWindows() ? commandRoot + ".exe": commandRoot; //$NON-NLS-1$
+		final String javaHomeString = System.getProperty("java.home");
+		File javaHome = new File(javaHomeString);
+		File javaHomeBin = new File(javaHome, "bin");
+		File  commandFile = new File(javaHomeBin, commandWithSuffix);
+		if (!commandFile.exists()) { /* Java 8 has 2 bin directories, and JAVA_HOME is probably pointing at jre */
+			log("Did not find " + commandFile.getAbsolutePath()); //$NON-NLS-1$
+			File javaHomeBinParentBin = new File(javaHomeBin.getParentFile(), "bin");
```

Comment:
This is the same as `javaHome`; I think you meant (also notice the variable name change):
> File javaHomeParentBin = new File(javaHome.getParentFile(), "bin");

Reply:
Yup.  Found that in testing.